Семинар_7q/Task_7-1.py

```diff
@@ -0,0 +1,12 @@
+stix = list(input().split())
```

Comment:
Метод `split` уже возвращает список.
```suggestion
stix = input().split()
```